src/index.js

```diff
@@ -57,16 +57,18 @@ const exported = {
      * used, must be set before `prewarm()` is called to have an effect.
      *
      * By default, the lifecycle of these resources is managed automatically, and they are
-     * lazily initialized when a Map is first created. By invoking `prewarm()`, these
-     * resources will be created ahead of time, and will not be cleared when the last Map
-     * is removed from the page. This allows them to be re-used by new Map instances that
+     * lazily initialized when a `Map` is first created. By invoking `prewarm()`, these
+     * resources will be created ahead of time, and will not be cleared when the last `Map`
+     * is removed from the page. This allows them to be re-used by new `Map` instances that
      * are created later. They can be manually cleared by calling
-     * `mapboxgl.clearPrewarmedResources()`. This is only necessary if your web page remains
-     * active but stops using maps altogether.
+     * `mapboxgl.clearPrewarmedResources()`. This is only necessary if your web page
+     * remains active but stops using maps altogether. `prewarm()` is idempotent
+     * and has guards against being executed multiple times, and any resources
+     * called by `prewarm()` are called synchronously.
```

Comment:
Perhaps emphasize the _creation_ of resources for clarity:

```suggestion
     * allocated by `prewarm()` are created synchronously.
```